src/Sylius/Bundle/PayumBundle/Action/CapturePaymentAction.php

```diff
@@ -67,7 +67,8 @@ public function execute($request)
                 $payumPayment->setClientId($order->getCustomer()->getId());
                 $payumPayment->setDescription(sprintf(
                     'Payment contains %d items for a total of %01.2f',
-                    $order->getItems()->count(), $totalAmount
+                    $order->getItems()->count(),
+                    $totalAmount
```

Comment:
Should be `round($totalAmount / 100, 2)`, right now for amount $12.00 it will display `... for a total of 1200.00` because the total is integer.